src/mac/mac_bs.h

```diff
@@ -4,102 +4,106 @@
  * Copyright (C) 2020 Lukas Ostendorf <lukas.ostendorf@gmail.com>
  *                    and the project contributors
  *
- * This library is free software; you can redistribute it and/or modify it under the terms of the
- * GNU Lesser General Public License as published by the Free Software Foundation; version 3.0.
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; version 3.0.
  *
- * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
- * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
- * See the GNU Lesser General Public License for more details.
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
  *
- * You should have received a copy of the GNU Lesser General Public License along with this library;
- * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
- * Boston, MA 02110-1301 USA
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
 #ifndef MAC_MAC_BS_H_
 #define MAC_MAC_BS_H_
 
+#include "mac_common.h"
 #include "mac_config.h"
 #include "mac_fragmentation.h"
-#include "mac_common.h"
 #include "tap_dev.h"
 
+#include "../phy/phy_bs.h"
```

Comment:
Re-arranging the includes in mac_bs.h and phy_bs.h causes compilation errors.

mac_bs.h needs some constants from the phy layer (e.g. FRAME_LEN), i would suggest to explicitly include them here

Add the following include 
+#include "../phy/phy_config.h"